.github/workflows/deploy-personal-dev-environment.yml

```diff
@@ -0,0 +1,159 @@
+name: Deploy Personal Dev Environment
+
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: "The name of the user on your machine e.g. janedoe"
+        required: true
+        type: string
+
+env:
+  AWS_REGION: "eu-west-2"
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
```

Comment:
This workflow will:
1. Deploy the infra
2. Pull images from the central ECR and push those images to the ECRs in this ne env
3. Restart the services so they are running the latest images
4. Bootstrap the database with the truncated test dataset

Right now the bootstrap job is fire and forget so I couldn't implement a follow up step to flush caches but once I can test this workflow and see it in action I'll come back to sticking the cache flush at the end